src/interfaces/Chain.ts

```diff
@@ -0,0 +1,8 @@
+import { Contract } from './Contract';
```

Comment:
FYI @Frenkiee I moved things inside the interfaces folder, I think it might be much easier to share the interfaces across things ( apis / components / utils etc. ) avoiding some circular deps later on